analyzers/tests/SonarAnalyzer.TestFramework/Verification/Verifier.cs

```diff
@@ -99,21 +99,7 @@ public Verifier(VerifierBuilder builder)
             }
             foreach (var compilation in Compile(builder.ConcurrentAnalysis))
             {
-                if (builder.IsRazor)
-                {
-                    DiagnosticVerifier.VerifyRazor(
-                        compilation,
-                        analyzers,
-                        builder.ErrorBehavior,
-                        builder.AdditionalFilePath,
-                        onlyDiagnosticIds,
-                        builder.Paths.Where(builder.IsRazorOrCshtmlFile).Select(TestCasePath),
-                        builder.Snippets.Where(x => builder.IsRazorOrCshtmlFile(x.FileName)));
```

Comment:
This is the root of the problem.
I also removed the `DiagnosticVerifier.VerifyRazor` on the way, as it was not really needed.